parsec_linux/pubspec.yaml

```
@@ -0,0 +1,32 @@
+name: parsec_linux
+description: Linux implementation of the parsec plugin.
+version: 0.1.0
+homepage:
+publish_to: none
+
+environment:
+  sdk: ">=2.18.5 <3.0.0"
+  flutter: ">=2.5.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+  parsec_platform_interface:
+    path: ../parsec_platform_interface/
```

```suggestion
  parsec_platform_interface: 0.1.0
```
we can already use the [pub.dev](https://pub.dev/packages/parsec_platform_interface) version, but it will be necessary to update it with the `parsec_eval_exception` file that you created. 

Let's just keep that in mind for the next PR.